.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+# General Information
+
+1. **What change/s does this pull request introduce? (Provide a detailed explanation)**
+
+2. **Does this pull request break something? What?**
+
+# Testing
```

Comment:
Could we update this with entries on unit-tests and e2e tests?

Reply:
Like `Was this unit tested?` or something like that? I didn´t add them because CI already runs the tests.